### PR TITLE
Font changes + title removal

### PR DESCRIPTION
**Pull request by jonocairns**
Suggesting some font changes along with some general UX fixes :)

- added global font and increased default font size (I'm not tied to this font but it looks cool 💃) [font link here](https://fonts.google.com/specimen/Montserrat)
- added descriptive placeholders for search and request input elements
- removed core pages H1 elements. Reasoning: the user already knows what page they're on from the navigation. I don't think we need the extra 50px cost of displaying the title again.
- fixed issue where the cursor would not be a pointer on the requests page move/tv tabs

image

### CODE REVIEW

File: src/Ombi/ClientApp/app/requests/movierequests.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 <div class="form-group">
     <div class="input-group">
-        <input type="text" id="search" class="form-control form-control-custom searchwidth" placeholder="Search" (keyup)="search($event)">
+        <input type="text" id="search" class="form-control form-control-custom searchwidth" placeholder="search for an existing movie request here..." (keyup)="search($event)">
```
Comment (tidusjar) [inline src/Ombi/ClientApp/app/requests/movierequests.component.html]: Translation for the placeholder?

File: src/Ombi/ClientApp/app/search/moviesearch.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<!-- Movie tab -->
 <div role="tabpanel" class="tab-pane active" id="MoviesTab">
     <div class="input-group">
-        <input id="search" type="text" class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
+        <input id="search" type="text" placeholder="search for a movie here..." class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
```
Comment (tidusjar) [inline src/Ombi/ClientApp/app/search/moviesearch.component.html]: Same as above regarding translations

**Comment by jonocairns** [comment]
@tidusjar I'm not too sure I've used the right structure for the translation files but I gave it a shot :D

**Comment by tidusjar** [comment]
Thanks, I'll pull the branch out to night and Chexk it out. See if the bee font works

File: src/Ombi/wwwroot/translations/de.json [hunk 3]
```diff
@@ -82,7 +82,8 @@
       "TopRatedMovies": "Am besten bewertete Filme",
       "NowPlayingMovies": "Im Kino",
       "HomePage": "Startseite",
-      "Trailer": "Trailer"
+      "Trailer": "Trailer",
+      "PlaceholderText": "search for a movie here..."
```
Comment (tidusjar) [inline src/Ombi/wwwroot/translations/de.json]: This text has now been duplicated a lot! That means there will be more than 1 potential translation for it, Maybe if we movie this and the other duplicates under the `"Common"` section

Comment (jonocairns) [reply src/Ombi/wwwroot/translations/de.json]: they aren't technically duplicates. one is the placeholder for tv/movie and the other is for tv/movie _requests_ 

```
"PlaceholderText": "search for a movie here..."
"PlaceholderText": "search for an existing movie request here..."

"PlaceholderText": "search for an existing tv show request here..."
"PlaceholderText": "search for a tv show here..."

```

**Comment by tidusjar** [comment]
Hey thanks for the PR by the way, but i'm probably not going to merge these in. I'm not sure I like the font and I think the heading should be there :)

**Pull request closed without merging**